src/pages/partners/_deriv-ib-programme.js

```diff
@@ -36,9 +32,8 @@ const BackButton = styled(Button)`
     color: var(--color-black);
 `
 const StyledText = styled(Text)`
-    font-size: ${props => (props.font_size ? props.font_size : '1.4rem')};
-    line-height: ${props => (props.lh ? props.lh : '1.15')};
-    margin-bottom: ${props => props.margin_bottom || ''};
+    font-size: 1.4rem;
+    line-height: 1.15;
```

Comment:
we have replicated the line-height in this component, in line 469 we declare it inline too.